src/heuristics.jl

```diff
@@ -1,3 +1,8 @@
+"""
```

Comment:
no need for a docstring at the file level, it should usually be clear from the name of the file. For the TODO, better have it as an issue